main.go

```diff
@@ -1451,7 +1570,11 @@ func unstageChanges(c *cli.Context) {
 
 // CLI function call - Generates automated commit
 func autoUpdate(c *cli.Context) {
-	generateChanges(true, false)
+	icons := c.Bool("icons")
+	// generateChanges(true, false)
```

Comment:
I think this shouldn't be commented out?

Reply:
Fixing!